app/models/school.rb

```diff
@@ -13,65 +15,31 @@ def has_active_survey?
 
   private 
 
-  def walkshed_generator
-    '''
-        WITH paths as (#{})
-        SELECT ST_AsEWKT(
-            ST_MakeValid(
-                ST_Transform(
-                    ST_Union(
-                        array(
-                            select ST_BUFFER(geometry, 100) from (#{}) as BIKE
-                        )
-                    ),
-                    26986
-                )
-            )
-        ) as _20,
-        ST_AsEWKT(
-            ST_MakeValid(
-                ST_Transform(
-                    ST_Union(
-                        array(
-                            select ST_BUFFER(geometry, 100) from paths where cost < 1.5
-                        )
-                    ),
-                    26986
-                )
-            )
-        ) as _15,
-        ST_AsEWKT(
-            ST_MakeValid(
-                ST_Transform(
-                    ST_Union(
-                        array(
-                            select ST_BUFFER(geometry, 100) from paths where cost < 1.0
-                        )
-                    ),
-                    26986
-                )
-            )
-        ) as _10,
-        ST_AsEWKT(
-            ST_MakeValid(
-                ST_Transform(
-                    ST_Union(
-                        array(
-                            select ST_BUFFER(geometry, 100) from paths where cost < 0.5
-                        )
-                    ),
-                    26986
-                )
-            )
-        ) as _05
-    '''
-  end
+    def update_sheds
+      # TODO: active job or sidekick or rabbit mq
+      sheds = WalkshedQuery.new(id).execute
 
-  def now
-    DateTime.now
-  end
+      # adds sheds to columns
+      update_columns({  shed_05: sheds[0]['_05'],
+                        shed_10: sheds[0]['_10'],
+                        shed_15: sheds[0]['_15'],
+                        shed_20: sheds[0]['_20'] })
+
+      # converts sheds into rings
+      shed_10_ring = shed_10.difference(shed_05).to_s
+      shed_15_ring = shed_15.difference(shed_10).to_s
+      shed_20_ring = shed_20.difference(shed_15).to_s
+
+      # updates with new rings
+      update_columns({  shed_10: shed_10_ring,
+                        shed_15: shed_15_ring,
+                        shed_20: shed_20_ring  })
+      
+    end
+
+    def now
```

Comment:
This method does not appear to be used? It's probably ok to explicitly call DateTime.now without encapsulating in a method anyways.